io.catenax.mp_standard_response/1.0.0/ProvideProductionForecast.ttl

```diff
@@ -0,0 +1,166 @@
+   samm:description "the concrete information about a production forecast"@en ;
+   samm:characteristic :ProductionForecastCharacteristic .
+
+:cxHeader a samm:Property ;
+   samm:preferredName "cxHeader"@en ;
+   samm:description "header that refers to the request of a customer"@en ;
+   samm:characteristic :CXHeaderCharacteristic .
+
+:ProductionForecastCharacteristic a samm:Characteristic ;
+   samm:preferredName "ProductionForecastCharacteristic"@en ;
+   samm:description "All Data that is related to a production forecast"@en ;
+   samm:dataType :ProductionForecastItem .
+
+:CXHeaderCharacteristic a samm:Characteristic ;
+   samm:preferredName "CXHeaderCharacteristic"@en ;
+   samm:description "all information required for a header specified by CX"@en ;
+   samm:dataType ext-header:MessageHeader .
+
+:ProductionForecastItem a samm:Entity ;
+   samm:preferredName "ProductionForecastItem"@en ;
+   samm:description "Datamodel of the concrete production forecast"@en ;
+   samm:properties ( :listOfForecastItems [ samm:property :iterationNo; samm:optional true ] :communicationMode ) .
+
+:listOfForecastItems a samm:Property ;
+   samm:preferredName "listOfForecastItems"@en ;
+   samm:description "List of ForecastItems matching the items to an order"@en ;
+   samm:characteristic :ForecastItemList .
+
+:iterationNo a samm:Property ;
+   samm:preferredName "iterationNo"@en ;
+   samm:description "only set in CommunicationMode == \"notification/cycle\" to be able to check the order of the notifications. "@en ;
+   samm:characteristic :IntegerCharacteristic .
+
+:communicationMode a samm:Property ;
+   samm:preferredName "communicationMode"@en ;
+   samm:description "confirmation of the requested communication mode"@en ;
+   samm:characteristic :CommunicationModeEnum ;
+   samm:exampleValue "synchronous" .
+
+:ForecastItemList a samm-c:List ;
+   samm:preferredName "ForecastItemList"@en ;
+   samm:description "List with the forecast items"@en ;
+   samm:dataType :ForecastItem .
+
+:IntegerCharacteristic a samm:Characteristic ;
+   samm:preferredName "IntegerCharacteristic"@en ;
+   samm:dataType xsd:integer .
+
+:CommunicationModeEnum a samm-c:Enumeration ;
+   samm:preferredName "CommunicationModeEnum"@en ;
+   samm:description "Enum that features all possible communication modes"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Month" "Day" "Week" "Hour" "Minute" "Second" ) .
+
+:ForecastItem a samm:Entity ;
+   samm:preferredName "ForecastItem"@en ;
+   samm:description "a ForecastItem entry for the requested order"@en ;
+   samm:properties ( :positionId :productionForecast :precisionOfForecast :prodcutionStatus [ samm:property :reasons4Delay; samm:optional true ] :returnCode :forecastDate ) .
+
+:positionId a samm:Property ;
+   samm:preferredName "positionId"@en ;
+   samm:description "Identifier of a position of an order"@en ;
+   samm:characteristic :String ;
+   samm:exampleValue "00000000-0000-0000-C000-000000000046" .
```

Comment:
Is this a pre-defined structure of the id? If so consider providing a regex for that